packages/taquito-signer/src/derivation-tools/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { InvalidHexStringError } from "@taquito/utils";
+import { InvalidHexStringError } from '../errors';
```

Comment:
I see this happening in a few files where a common error is being imported from `@taquito/core` and is re-exported into the file that needs it. I think for those cases we should instead import directly from `@taquito/core`.

The only case where we'd need to import and re-export is when the error used to live in the package, but is then moved into `@taquito/core`. We should be leveraging direct imports from `@taquito/core` wherever possible

Reply:
good point i'll make sure to change them import directly from taquito/core
just wanted to be consistent and picked the import path that was longer 😓